config/jobs/kubernetes/sig-testing/make-test.yaml

```diff
@@ -58,8 +58,7 @@ periodics:
             - |
               result=0
               # Run the tests as usual
-              ( cd hack/tools && GO111MODULE=on go install gotest.tools/gotestsum ) || result=$?
```

Comment:
Don't we still need this ?

Reply:
nvm just saw the commit description

/hold cancel

Reply:
liggitt fixed this when we started migrating things to `make test`, it does the sane thing and installs a copy from hack/tools/go.mod's version into a local GOBIN and uses that instead of depending on one in system PATH

Reply:
(in 1.21 devel)